python/pyspark/sql/cogroup.py

```diff
@@ -0,0 +1,98 @@
+#
```

Comment:
Seems like we don't generate documentation for this:

![Screen Shot 2019-09-22 at 9 41 48 PM](https://user-images.githubusercontent.com/6477701/65387377-d96f9900-dd81-11e9-928f-de9929571355.png)

cannot click.

It should be either documented at `python/docs/pyspark.sql.rst` or imported at `pyspark/sql/__init__.py` with including it at `__all__`.

Reply:
+1 adding it to `pyspark/sql/__init__.py` with including it at `__all__` since this is what `group.py` does